src/locales/en/common.json

```diff
@@ -1,4 +1,49 @@
 {
   "send": "send",
```

Comment:
S in send should be capitalized: `"send": "Send",` It's the same on the button, but different as the send modal title.